doc/complete-reference.md

```diff
@@ -84,8 +84,28 @@ Nelmio\Entity\User:
         favoriteNumber: 42
 ```
 
-To go further we can just randomize data.
+You can refer to previous fixtures to get the same amount of fixtures.
+The current formatter is set to the referenced fixture.
 
+```yaml
+Nelmio\Entity\UserDetail:
+# reference by concrete id
+    user{@user1}:
```

Comment:
It is more effective for the use with a mask where you can get a list of objects. If you use a concrete reference of an object you could easily establish 1:1 associations. 

A example would be like you have a car:

`\Car:`
`  car_{1..100}`
`    general_car information...`

`\CarMotor:`
`  carmotor_{1..100}: <-- you need to specify the exact amount of cars in the above example`
`    car: '@car_<current()>'`
`    motor_information...`

My solution would be:

`\CarMotor:`
`  carmotor_{@car_*}: <-- gets your previously declared amount of objects`
`    car: <current()> <-- has the current car as current-reference`
`    motor_information....`

As every car have a motor you don't need to reference the exact number, you could just change it to 500 cars and you still have 500 motor information. 

Hope this makes it a bit clearer, if not feel free to ask.